content/docs/command-reference/exp/remove.md

```diff
@@ -85,9 +85,9 @@ $ dvc exp remove e41d5b4 split16
 $ dvc exp show --include-params=train.min_split --no-pager
 ```
 
-```table
+```dvctable
 ┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment            ┃ Created      ┃ State  ┃ avg_prec ┃ roc_auc ┃ train.min_split ┃
+┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃ neutral:**State**  ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ metric:**train.min_split** ┃
```

Comment:
> should we highlight the header (colorize params, metrics, etc)?

Sanity checked in https://dvc-org-ref-dvctable-ojpnqwjsg.herokuapp.com/doc/command-reference/exp/remove#examples:

![image](https://user-images.githubusercontent.com/1477535/131789448-492768b8-cf2b-4136-9751-bb979e877976.png)

Reply:
The scroll bar isn't ideal but it's a problem in most `exp show` tables now (it looked better when they were images but wasn't copy-pasteable).